src/frontends/pytorch/src/op/is_nonzero.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/not_equal.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_is_nonzero(const NodeContext& context) {
+    num_inputs_check(context, 1, 1);
+    auto input = context.get_input(0);
+
+    Output<Node> zero_tensor = context.mark_node(v0::Constant::create(element::boolean, Shape{1}, {false}));
+
+    align_eltwise_input_types(context, input, zero_tensor);
```

Comment:
Please use `ConvertLike` instead of `align_eltwise_input_types`. It is not required here.